src/adapter/graphql/src/scalars/odf_generated.rs

```diff
@@ -85,8 +85,8 @@ pub struct AttachmentEmbedded {
 impl From<odf::AttachmentEmbedded> for AttachmentEmbedded {
     fn from(v: odf::AttachmentEmbedded) -> Self {
         Self {
-            path: v.path.into(),
-            content: v.content.into(),
+            path: v.path,
```

Comment:
Note that you are changing a code-generated file

Reply:
I'll exclude this file for now -- a good catch! :medal_sports: